### ISSUE TITLE
复现其他数据集上的数据时，参数怎么设置？

### ISSUE THREAD

**Issue by ykf173** [ShannonAI/glyce#23]
您好，在复现其它任务时，遇到的对于其他任务的以下的参数怎么设置，比如说ner的微博数据等？
max_seq_len=150
learning_rate=3e-5
warmup=0.2
local_rank=-1
seed=3307
checkpoint=100
